src/general_manager/bucket/databaseBucket.py

```
@@ -94,33 +95,104 @@ def __mergeFilterDefinitions(
+            **kwargs
         )
+        qs = self._data
+        if annotations:
+            other_annotations: dict[str, Any] = {}
+            for key, value in annotations.items():
+                if callable(value):
+                    other_annotations[key] = value
+                    continue
+                qs = value(qs)
+            if not isinstance(qs, models.QuerySet):
+                raise TypeError("Query annotation must return a Django QuerySet")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)